tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,4769 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.                             .str.replace(PUNCTUATIONS, [" "]*len(PUNCTUATIONS), regex=False)
</pre><p> </p><p>Why does  	<code>[" "]</code>  have to be a list ? </p><p>Also, <code>replace_tokens</code>   might be faster here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>